packages/subiquity_client/lib/src/types.dart

```diff
@@ -254,10 +254,10 @@ class StorageResponse with _$StorageResponse {
 @freezed
 class WSLConfigurationBase with _$WSLConfigurationBase {
   const factory WSLConfigurationBase({
-    @JsonKey(name: 'custom_path') String? customPath,
-    @JsonKey(name: 'custom_mount_opt') String? customMountOpt,
-    @JsonKey(name: 'gen_host') bool? genHost,
-    @JsonKey(name: 'gen_resolvconf') bool? genResolvconf,
+    @JsonKey(name: 'automount_root') String? automountRoot,
+    @JsonKey(name: 'automount_options') String? automountOptions,
+    @JsonKey(name: 'network_generatehosts') bool? networkGeneratehosts,
+    @JsonKey(name: 'networkGenerateresolvconf') bool? networkGenerateresolvconf,
```

Comment:
network_generateresolvconf

Reply:
Actually, screw that! I have it on my TODO-list to make all these use automatic snake_case renames. We could go straight for this to let the JSON serializer do it for us:
```dart
@freezed
class WSLConfigurationBase with _$WSLConfigurationBase {
  @JsonSerializable(fieldRename: FieldRename.snake)
  const factory WSLConfigurationBase({
    String? automountRoot,
    String? automountOptions,
    bool? networkGeneratehosts,
    bool? networkGenerateresolvconf,
  }) = _WSLConfigurationBase;

  factory WSLConfigurationBase.fromJson(Map<String, dynamic> json) =>
      _$WSLConfigurationBaseFromJson(json);
}
```